presto-main/src/main/java/io/prestosql/sql/analyzer/OutputTable.java

```diff
@@ -23,14 +23,14 @@
 import static java.util.Objects.requireNonNull;
 
 @Immutable
-public final class Output
+public final class OutputTable
```

Comment:
What's the purpose of this rename? I don't see the benefit in changing it.

Reply:
I wanted to distinguish `targetTable` from `targetSchema`. But maybe renaming the class was too much.